pyproject.toml

```diff
@@ -35,7 +35,8 @@ Releases = "https://nvidia.github.io/cuda-quantum/latest/releases.html"
 # We must use h5py<3.11 because 3.11 doesn't include aarch64 Linux wheels.
 # https://github.com/h5py/h5py/issues/2408
 [project.optional-dependencies]
-chemistry = [ "scipy==1.10.1", "openfermionpyscf==0.5", "h5py<3.11" ]
+chemistry = [ "scipy==1.10.1", "openfermionpyscf==0.5", "h5py<3.11"  ]
+visualization = [ "qutip>=5" ]
```

Comment:
Hi, I've added Matplotlib as a dependency here. 
Normally it is installed while installing qutip, but it may not get installed in some edge case or some minimal system, so I had not put it in. 😅